### DISCUSSION TITLE
How to install spectral with curl on an arm64 architecture machine

### DISCUSSION THREAD

**Discussion by Jeremy-boo** [stoplightio/spectral#1892] [Q&A]
Hello, I have a question about how to install spectral with curl on an arm64 architecture machine,Do you have any suggestions of how can I do it? From what I know so far, spectral does not have an arm64 binary package yet.

**Comment by P0lip** [comment]
Hey!
Unfortunately, at the time being, we don't distribute ARM binaries.
I believe one may be able to build them themselves.
```shell
git clone https://github.com/stoplightio/spectral.git
yarn
yarn build
yarn workspace @stoplight/spectral-cli pkg . --public --targets node16-linux-arm64 --output ./binaries/spectral
````
If the build succeeds, the binaries can be found in `packages/cli/binaries`.
For the list of available targets, please refer to [pkg](https://github.com/vercel/pkg#targets).

The example above assumes you have Node.js & Yarn installed.

**Reply by Jeremy-boo** [reply]
Ok，I'll try this way,  thank you very much!
